9.1 HW Yuri Shipitsyn.txt

```diff
@@ -0,0 +1,208 @@
+
+Partition table scan:
+  MBR: protective
+  BSD: not present
+  APM: not present
+  GPT: present
+
+Found valid GPT with protective MBR; using GPT.
+
+Command (? for help): p #Checking what i have created
+Disk /dev/sdb: 10485760 sectors, 5.0 GiB
+Logical sector size: 512 bytes
+Disk identifier (GUID): B99CC69F-94E0-454C-8FA8-E1701E620E5A
+Partition table holds up to 128 entries
+First usable sector is 34, last usable sector is 10485726
+Partitions will be aligned on 2048-sector boundaries
+Total free space is 5242813 sectors (2.5 GiB)
+
+Number  Start (sector)    End (sector)  Size       Code  Name
+   1            2048         4196351   2.0 GiB     8300  Linux filesystem
+   2         4196352         5244927   512.0 MiB   8200  Linux swap
+
+1.3. Format the 2GB partition with an XFS file system
+
+[root@localhost ~]# mkfs.xfs -f -L XFS /dev/sdb1
+meta-data=/dev/sdb1              isize=512    agcount=4, agsize=131072 blks
+         =                       sectsz=512   attr=2, projid32bit=1
+         =                       crc=1        finobt=0, sparse=0
+data     =                       bsize=4096   blocks=524288, imaxpct=25
+         =                       sunit=0      swidth=0 blks
+naming   =version 2              bsize=4096   ascii-ci=0 ftype=1
+log      =internal log           bsize=4096   blocks=2560, version=2
+         =                       sectsz=512   sunit=0 blks, lazy-count=1
+realtime =none                   extsz=4096   blocks=0, rtextents=0
+
+1.4. Initialize 512MB partition as swap space
+
+[root@localhost ~]# mkswap /dev/sdb2 #Initializing swap
+Setting up swapspace version 1, size = 524284 KiB
+no label, UUID=c93bb97f-b8f1-45ff-91b0-e440f3d6ca5b
+[root@localhost ~]# swapon /dev/sdb2 #Enabling swap
+
+1.5  Configure the newly created XFS file system to persistently mount at /backup
+
+[root@localhost ~]# blkid #To receive info about our disc names
+/dev/sda1: UUID="afd305e0-cc70-4cc5-860e-44cf9f065d1e" TYPE="xfs"
+/dev/sda2: UUID="i3GmTc-xAfJ-AEwE-AI3w-35pl-9H8N-8U1c13" TYPE="LVM2_member"
+/dev/sdb1: LABEL="XFS" UUID="e2ec48a4-6bf7-49fd-a79f-8e7301315632" TYPE="xfs" PARTLABEL="Linux filesystem" PARTUUID="c502c531-c7d3-4680-8dbb-c3045361ecbb"
+/dev/sdb2: UUID="c93bb97f-b8f1-45ff-91b0-e440f3d6ca5b" TYPE="swap" PARTLABEL="Linux swap" PARTUUID="407c3b3e-709d-4859-b59a-05049c4f8763"
+/dev/mapper/centos-root: UUID="1ed72b01-66ba-4a8b-8253-3594709bad3a" TYPE="xfs"
+/dev/mapper/centos-swap: UUID="1bc376b3-12e0-4511-8ad2-097dedea8ec4" TYPE="swap"
+
+[root@localhost ~]# mkdir /backup
+[root@localhost ~]# mount /dev/sdb1 /backup
+#Uptading /etc/fstab to use swap after a system reboot
+/dev/sdb1 	/backup		xfs	defaults	0 0
+
+1.6. Configure the newly created swap space to be enabled at boot
+
+#Uptading /etc/fstab to use swap after a system reboot
+/dev/sdb2       swap       swap       defaults       0 0
+
+1.7  Reboot your host and verify that /dev/sdc1 is mounted at /backup and that your swap partition  (/dev/sdc2) is enabled

```

Comment:
It would be nice to see here a command that will verify /etc/fstab has correct syntax before doing reboot